pkgs/initiator/initiator.go

```diff
@@ -212,82 +212,143 @@ func (c *Initiator) initMessageFlowHandling(init *spec.Init, id [24]byte, operat
 	return finalResults, nil
 }
 
-func (c *Initiator) ResignMessageFlowHandling(rMsg *wire.SignedResign, id [24]byte, operators []*spec.Operator) ([][]byte, error) {
-	resignResult, errs, err := c.SendResignMsg(id, rMsg, operators)
+func (c *Initiator) ResignMessageFlowHandling(signedResign *wire.SignedResign, id [24]byte, operators []*spec.Operator) ([][][]byte, error) {
+	// reqIDtracker is used to track if all ceremony are in the responses in the expected order
+	reqIDs := make([][24]byte, 0)
+	for _, msg := range signedResign.Messages {
+		reqID, err := utils.GetReqIDfromMsg(msg)
+		if err != nil {
+			return nil, err
+		}
+		reqIDs = append(reqIDs, reqID)
+	}
+	resignResult, errs, err := c.SendResignMsg(id, signedResign, operators)
```

Comment:
does the initiator also validate the signature of the message before sending? Or you decided it isn't neccessary?

Reply:
I think it is not necessary, the operators will validate it. So if it is invalid it fails anyway.